cli/admin/call.go

```diff
@@ -207,5 +207,5 @@ func printMessage(pushedData interface{}) {
 		return
 	}
 
-	log.Info(msg)
+	fmt.Print(msg)
```

Comment:
Why not `Println()`?

Reply:
Yes you're right, but it would severely affect code already written (as opposed to removing points), so I'd rather not change the behavior at that point.

Reply:
OK, no problem :)

Reply:
done